addons/hr_attendance/__init__.py

```diff
@@ -1,5 +1,12 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import controllers
 from . import models
+
+
+def post_init_hook(env):
+    env['res.company']._check_hr_presence_control(True)
+
+
```

Comment:
Too much spacing
```suggestion

def post_init_hook(env):
    env['res.company']._check_hr_presence_control(True)

```

Reply:
Due to check style E302, we need to add two blank lines; otherwise, the check style will fail.